### PR TITLE
fully implement puppeteer@2.1.1/lib/BrowserFetcher.js 

### PR DESCRIPTION
**Pull request by Mattwmaster58**
Supersedes and closes #42

### CODE REVIEW

File: pyppeteer/browser_fetcher.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Chromium download module."""
```
Comment (Granitosaurus) [inline pyppeteer/browser_fetcher.py]:
```suggestion
"""
Chromium download module.

Chromium is being downloaded from:
https://commondatastorage.googleapis.com/chromium-browser-snapshots
see full download instructions:
https://www.chromium.org/getting-involved/download-chromium
"""
```

The chromium download instructions are awfully complicate - I'd like to keep this short docstring until we work on updating communication.

File: pyppeteer/browser_fetcher.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+NO_PROGRESS_BAR = bool(strtobool(os.environ.get('PYPPETEER2_NO_PROGRESS_BAR', 'false')))
+
+# chromiumExecutable = {
+#     'linux': DOWNLOADS_FOLDER / '{revision}' / 'chrome-linux' / 'chrome',
+#     'mac': (DOWNLOADS_FOLDER / REVISION / 'chrome-mac' / 'Chromium.app' /
+#             'Contents' / 'MacOS' / 'Chromium'),
+#     'win32': DOWNLOADS_FOLDER / REVISION / windowsArchive / 'chrome.exe',
+#     'win64': DOWNLOADS_FOLDER / REVISION / windowsArchive / 'chrome.exe',
+# }
```
Comment (Granitosaurus) [inline pyppeteer/browser_fetcher.py]:
```suggestion

```

Do we need this?

Comment (Mattwmaster58) [reply pyppeteer/browser_fetcher.py]: No, it's a left over artifact.

File: pyppeteer/browser_fetcher.py (hunk 1 above)
Comment (Mattwmaster58) [reply pyppeteer/browser_fetcher.py]: Looks like we're actually downloading from `https://storage.googleapis.com`.

**Pull request merged**